scripts/ci/lib/get-version.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -eu
```

Comment:
I think this script is a bit of a hack - all the github runners have `jq` - it might be better to use that.

(I think I wrote this script originally, sorry)

Reply:
It's ok man, I've updated it now to use `jq`  and added a dry run just so we can be sure on merge & publish that everything is kushdi